orient_express/vertex.py

```
@@ -0,0 +1,308 @@
+
+
+def upload_model(
+    model: Predictor,
+    model_name: str,
+    project_name: str,
+    region: str,
+    bucket_name: str,
+    serving_container_image_uri: str = "",
+    serving_container_health_route: str = "",
+    serving_container_predict_route: str = "",
+    labels: dict[str, str] | None = {},
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)